library/database/class.database.php

```diff
@@ -153,9 +153,8 @@ protected function newPDO($Dsn, $User, $Password) {
             $PDO = new PDO(strtolower($this->Engine).':'.$Dsn, $User, $Password, $this->ConnectionOptions);
             $PDO->setAttribute(PDO::ATTR_EMULATE_PREPARES, 0);
 
-            if ($this->ConnectionOptions[1002]) {
-                $PDO->query($this->ConnectionOptions[1002]);
-            }
+            $encoding = c('Database.CharacterEncoding', 'utf8mb4');
```

Comment:
I can move this into that one if you prefer: https://github.com/vanilla/vanilla/pull/5255 but the result will be the same. Using utf8mb4 for the connexion will cause no problem at all even if the client is using utf8.